docs_zh-CN/tutorials/customize_mot_model.md

```diff
@@ -1 +1,276 @@
+
+@REID.register_module()
+class MyReID(BaseModule):
+
+    def __init__(self,
+                arg1,
+                arg2):
+        pass
+
+    def forward(self, inputs):
+        # implementation is ignored
+        pass
+```
+
+#### 2. 引入模块
+
+你可以在 `mmtrack/models/reid/__init__.py` 中加入下面一行。
+
+```python
+from .my_reid import MyReID
+```
+
+或者，为了避免更改原始代码，你还可以在 config 文件中增加以下几行来实现：
+
+```python
+custom_imports = dict(
+    imports=['mmtrack.models.reid.my_reid.py'],
+    allow_failed_imports=False)
+
+```
+
+#### 3. 更改原始 config 文件
+
+```python
+motion=dict(
+    type='MyReID',
+    arg1=xxx,
+    arg2=xxx)
+```
+
+### 增加一个新的跟踪头
+
+#### 1. 定义一个跟踪头（例如：MyHead）
+
+新建一个文件 `mmtrack/models/track_heads/my_head.py`。
+
+``` python
+from mmcv.runner import BaseModule
+
+from mmdet.models import HEADS
+
+@HEADS.register_module()
+class MyHead(BaseModule):
+
+    def __init__(self,
+                arg1,
+                arg2):
+        pass
+
+    def forward(self, inputs):
+        # implementation is ignored
+        pass
+```
+
+#### 2. 引入模块
+
+你可以在 `mmtrack/models/track_heads/__init__.py` 中加入下面一行。
+
+```python
+from .my_head import MyHead
+```
+
+或者，为了避免更改原始代码，你还可以在 config 文件中增加以下几行来实现：
+
+```python
+custom_imports = dict(
+    imports=['mmtrack.models.track_heads.my_head.py'],
+    allow_failed_imports=False)
+```
+
+#### 3. 更改原始 config 文件
+
+```python
+motion=dict(
+    type='MyHead',
+    arg1=xxx,
+    arg2=xxx)
+```
+
+### 增加一个新的损失函数
+
+#### 1. 定义一个损失函数
+
+假定你想要增加一个新的损失函数 `MyLoss` 来进行边界框回归。为此，你需要定义一个文件 `mmtrack/models/losses/my_loss.py`。装饰器 `weighted_loss` 可以对损失函数输出结果做基于单个元素的加权平均。
+
+```python
+import torch
+import torch.nn as nn
+
+from mmdet.models import LOSSES, weighted_loss
+
+@weighted_loss
+def my_loss(pred, target):
+    assert pred.size() == target.size() and target.numel() > 0
+    loss = torch.abs(pred - target)
+    return loss
+
+@LOSSES.register_module()
+class MyLoss(nn.Module):
+
+    def __init__(self, reduction='mean', loss_weight=1.0):
+        super(MyLoss, self).__init__()
+        self.reduction = reduction
+        self.loss_weight = loss_weight
+
+    def forward(self,
+                pred,
+                target,
+                weight=None,
+                avg_factor=None,
+                reduction_override=None):
+        assert reduction_override in (None, 'none', 'mean', 'sum')
+        reduction = (
+            reduction_override if reduction_override else self.reduction)
+        loss_bbox = self.loss_weight * my_loss(
+            pred, target, weight, reduction=reduction, avg_factor=avg_factor)
+        return loss_bbox
+```
+
+#### 2. 引入模块
+
+你可以在 `mmtrack/models/losses/__init__.py` 中加入下面一行。
+
+```python
+from .my_loss import MyLoss, my_loss
+```
+
+或者，为了避免更改原始代码，你还可以在 config 文件中增加以下几行来实现：
+
+```python
+custom_imports=dict(
+    imports=['mmtrack.models.losses.my_loss'],
+    allow_failed_imports=False)
+```
+
+#### 3. 更改原始 config 文件
+
+假设 `MyLoss` 是用于回归任务，则在 `head` 区域中加入 `loss_bbox`。
```

Comment:
delete this line , and translate `To use it, modify the loss_xxx field. Since MyLoss is for regression, you need to modify the loss_bbox field in the head.`